index.html

```diff
@@ -302,6 +303,17 @@ <h2>
             A [=list=] containing the maximum logical value for each button
           </td>
         </tr>
+        <tr>
+          <td>
+            <dfn data-dfn-for="Gamepad">[[\touchEvents]]</dfn>
+          </td>
+          <td>
+            `null`
+          </td>
+          <td>
+            List of generated touch events.
```

Comment:
Ok, so... are these _really_ touch "events"? If we look at `GamepadTouch`, it's not an "Event" at all, so this is a misnomer. 

They are more like snapshots of touch points.